src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2278,6 +2302,15 @@ private sealed class PENamedTypeSymbolNonGeneric : PENamedTypeSymbol
             {
             }
 
+            private PENamedTypeSymbolNonGeneric(PENamedTypeSymbolNonGeneric other) :
```

Comment:
I don't think the purpose of this constructor is very obvious. Consider a doc comment, or taking `IsNativeInt` as a parameter?

Reply:
This is a temporary approach. I've added a comment.

---
In reply to: [352874410](https://github.com/dotnet/roslyn/pull/39157#discussion_r352874410) [](ancestors = 352874410)